src/__tests__/core/block.transactionFilter.test.ts

```diff
@@ -89,6 +89,78 @@ describe('transactionFilter on Block->transactions', async () => {
     expect(result.errors).toHaveLength(1);
     expect(result.errors[0].message).toMatch(/^Expected type TransactionFilter/);
   });
+
+  test('contractCreation: true value, returns only transactions with contract creations', async () => {
+    const query = `
+      {
+        block(number: 4180981) {
+          transactions(filter: { contractCreation: true }) {
```

Comment:
I would expect you to select the `to->address` field and assert that it is empty in all cases.

Reply:
SGTM